src/System.Data.Common/src/System/Data/SQLTypes/SQLBinary.cs

```diff
@@ -11,7 +11,6 @@
 
 namespace System.Data.SqlTypes
 {
-    [Serializable]
```

Comment:
Should we keep all of the SqlXx types?  Seems a bit odd to have only a subset of them.